django/core/handlers/base.py

```diff
@@ -105,6 +134,12 @@ def get_exception_response(self, request, resolver, status_code, exception):
         return response
 
     def get_response(self, request):
+        if settings.MIDDLEWARES is None:
```

Comment:
Would it be better to base this off of whether `self._middleware_chain` is populated, keeping all the settings-related logic encapsulated in `load_middleware`?